dev/com.ibm.ws.org.apache.cxf.cxf.rt.transports.http.3.2/src/org/apache/cxf/transport/http/Headers.java

```diff
@@ -0,0 +1,581 @@
+/**
```

Comment:
You should also add the Headers class from cxf 3.1.18 to com.ibm.ws.jaxrs.2.0.common just like you are doing here with the 3.3 version.